### ISSUE TITLE
about "self.scheduler.step() " and "self.loss.step()"

### ISSUE THREAD

**Issue by wanggrun** [seathiefwang/MGN-pytorch#18]
https://github.com/seathiefwang/MGN-pytorch/blob/384e82002e40b71157506e23d71e6709fa830757/trainer.py#L34

Hi, thank you for the nice repo!

In the 33rd and 34th line of trainer.py, there are two step functions, i.e.,  self.scheduler.step() and self.loss.step(). Could you explain the roles of these two functions?
